src/one_dragon_qt/overlay/overlay_hud_window.py

```
@@ -0,0 +1,324 @@
+            "fairyTriUrl",
+            QUrl.fromLocalFile(
+                os_utils.get_resource_path(
+                    "assets", "ui", "fairy_badge", "triangles.svg"
+                )
+            ),
+        )
+
+        self._engine.load(QUrl.fromLocalFile(str(self._qml_path)))
+        roots = self._engine.rootObjects()
+        if not roots:
+            raise RuntimeError("overlay_hud.qml 加载失败，无 root 对象")
```

Ruff (0.16.2)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)